tests/ui_tools/test_messages.py

```diff
@@ -945,13 +975,13 @@ def test_main_view_generates_PM_header(
     )
     def test_msg_generates_search_and_header_bar(
         self,
-        mocker,
-        messages_successful_response,
-        msg_type,
-        msg_narrow,
-        assert_header_bar,
-        assert_search_bar,
-    ):
+        mocker: MockerFixture,
+        messages_successful_response: Dict[str, Any],
+        msg_type: MessageType,
```

Comment:
Ah, I see you used `MessageType` here - but note that the variable type is different here, which I can tell from reading the parametrize (as per other comments).